src/Stratis.SmartContracts.Core/State/ContractStateRepository.cs

```diff
@@ -100,6 +100,19 @@ public byte[] GetStorageValue(uint160 addr, byte[] key)
             return accountState == null ? null : this.storageCache.Get(addr.ToBytes()).Get(key);
         }
 
+        public string GetContractType(uint160 addr)
+        {
+            AccountState accountState = this.GetAccountState(addr);
+            return accountState != null ? accountState.TypeName : "";
```

Comment:
"" to string.Empty.

```return accountState != null ? accountState.TypeName : string.Empty;```

Reply:
I have to ask if this is worth holding up a PR for? There is no difference between string.Empty and "" - they both refer to the same string literal. You can check this with `ReferenceEquals("", string.Empty);`.

In which case, this is a coding standards issue, and are we *really* at this level of finicky?

Reply:
I changed it for now but if that is actually written somewhere I'm really against it and will never enforce it in PRs I'm reviewing. 

More rules kills the coding flair yo!